test-cases/gutenberg/button.md

```diff
@@ -0,0 +1,196 @@
+-   Edit `Link Rel` field and add rel
+-   Open button [options](../resources/button-options.png)
+-   Expect fields `Button URL`, `Open in new tab`, `Link Rel` to have the same content as in link settings
+
+--------------------------------------------------------------------------------
+
+##### TC008
+
+### Clear link settings
+
+-   Add a Button block
+-   Open link [settings](../resources/button-link-settings.png)
+-   Edit `Button URL` field and add link
+-   Switch on `Open in new tab`
+-   Edit `Link Rel` field and add rel
+-   Press `Remove link` button
+-   Open link settings again
+-   Expect default settings
+
+--------------------------------------------------------------------------------
+
+##### TC009
+
+### Change Button border radius
+
+-   Add a Button block
+-   Open button [options](../resources/button-options.png)
+-   Edit `Border Radius` moving slider
+-   Expect Button border radius changed accrodingly to the value
+-   Edit `Border Radius` typing border radius value in input
+-   Expect Button border radius changed accordingly to the value
+
+--------------------------------------------------------------------------------
+
+##### TC010
+
+### Render custom background color
+
+-   Open post on web
+-   Add a Button block
+-   Choose the **custom** background color from **picker**
+-   Save the post
+-   Open the same post on mobile app
+-   Expect Button background color is the same as on web
+
+--------------------------------------------------------------------------------
+
+##### TC011
+
+### Render custom text color
+
+-   Open post on web
+-   Add a Button block
+-   Choose the **custom** text color from **picker**
+-   Save the post
+-   Open the same post on mobile app
+-   Expect Button text color is the same as on web
+
+--------------------------------------------------------------------------------
+
+##### TC012
+
+### Render gradient background color
+
+-   Open post on web
+-   Add a Button block
+-   Choose the **gradient** background color from the palette
+-   Save the post
+-   Open the same post on mobile app
+-   Expect Button gradient background color is the same as on web
+
+--------------------------------------------------------------------------------
+
+##### TC013
+
+### Render fallback background and text colors
+
+-   Open post on web
+-   Add a Button block
+-   Choose the **regular** background color from the palette
+-   Choose the **regular** text color from the palette
+-   Save the post
+-   Open the same post on mobile app
+-   Expect Button with gray background color and white text color
+
+--------------------------------------------------------------------------------
+
+##### TC014
+
+### Render blue background color for new Button
+
+-   Add a Button block
+-   Expect blue background color and white text
+
+--------------------------------------------------------------------------------
+
+##### TC015
+
+### Render selection / caret color only for custom colors
```

Comment:
I think we don't need to emphasize "only for custom colors" since it is not a feature, this will be true for theme colors as well once they are available for us through an endpoint. The steps already have good info to test this properly.

```suggestion
### Check if selection / caret color matches font color
```